Resources/views/GraphiQL/index.html.twig

```diff
@@ -46,7 +46,7 @@
         var xdebugToken = res.headers.get('X-Debug-Token')
         if (typeof Sfjs !== "undefined" && xdebugToken) {
           var toolbarElement = document.querySelector('.sf-toolbar')
-          var debugUrlPattern = "{{ url('_wdt', {'token': '__TOKEN__'}) }}"
+          var debugUrlPattern = "{{ app.debug ? url('_wdt', {'token': '__TOKEN__'}) : '/_wdt/__TOKEN__' }}"
```

Comment:
maybe the best way is to remove the `if` condition if not in debug mode? so doing we don't keep useless condition.
what you think of that?